docs/reference/cluster/stats.asciidoc

```diff
@@ -38,6 +38,278 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=flat-settings]
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
 
+[[cluster-stats-api-response-body]]
+==== {api-response-body-title}
+
+`_nodes.total`::
+(integer)
+Total number of nodes in the cluster.
```

Comment:
The `GET /_cluster/stats/nodes/<node_id>` form of this API call is not documented correctly - in fact it takes a [list of node filters](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster.html#cluster-nodes) similarly to many other cluster APIs:

```
GET /_cat/nodes

# 127.0.0.1 15 100 6 2.48   ilm * node-0
# 127.0.0.1 32 100 6 2.48   dil - node-4
# 127.0.0.1 22 100 6 2.48   ilm - node-2
# 127.0.0.1 17 100 6 2.48   ilm - node-1
# 127.0.0.1 12 100 6 2.48   dil - node-3
#

GET /_cluster/stats/nodes/_all?filter_path=_nodes

# {
#   "_nodes": {
#     "successful": 5,
#     "total": 5,
#     "failed": 0
#   }
# }

GET /_cluster/stats/nodes/_master?filter_path=_nodes

# {
#   "_nodes": {
#     "successful": 1,
#     "total": 1,
#     "failed": 0
#   }
# }

GET /_cluster/stats/nodes/data:true?filter_path=_nodes

# {
#   "_nodes": {
#     "successful": 2,
#     "total": 2,
#     "failed": 0
#   }
# }

GET /_cluster/stats/nodes/nonexistent?filter_path=_nodes

# {
#   "_nodes": {
#     "successful": 0,
#     "total": 0,
#     "failed": 0
#   }
# }
```

Then the `_nodes.total` field is the total number of nodes selected by the node filters, which defaults to all nodes.

Reply:
Thanks so much @DaveCTurner. I corrected the request format and parm as well as this response parameter with 6eb0067471e.